src/components/icon-button/icon-button.style.js

```diff
@@ -27,6 +27,7 @@ const StyledIconButton = styled.button.attrs({ type: "button" })`
 
     ${StyledIcon} {
       ${disabled && `color: ${theme.icon.disabled}`};
+      ${disabled && `background-color: transparent`};
```

Comment:
```suggestion
      ${disabled && css`
          color: ${theme.icon.disabled};
          background-color: transparent;
      `};
```

Reply:
Good suggestion. I'll get it changed now 😄